src/index.js

```diff
@@ -13,30 +13,32 @@ import 'antd/dist/antd.less';
 import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
 import {
-  // CourseForm,
-  // CourseView,
-  // LandingPage,
-  // ModuleForm,
-  // ModuleView,
-  // ProgramForm,
-  // ProgramView,
+  CourseForm,
+  CourseView,
+  ModuleForm,
+  ModuleView,
+  SettingsPage,
+  ProgramForm,
+  ProgramView,
   HomePage,
   LoginPage,
   NotFoundPage,
 } from './components/pages';
 import {
```

Comment:
I appreciate that you pulled out the paths we use into one file and have imported them here. This will absolutely help cut out a large amount of human error! This is another great example of how you've thought of building these parts of the app with maintainability and readability in mind. Excellent.